types/inet.go

```diff
@@ -126,33 +126,39 @@ func (a InetArray) String() string {
 		}
 		switch {
 		case a.IsNull(i):
-			o.WriteString("(null)")
+			o.WriteString(array.NullValueStr)
 		default:
-			fmt.Fprintf(o, "\"%s\"", arr.Value(i))
+			fmt.Fprintf(o, "%q", a.ValueStr(i))
 		}
 	}
 	o.WriteString("]")
 	return o.String()
 }
 
-func (a *InetArray) ValueStr(i int) string {
+func (a *InetArray) Value(i int) *net.IPNet {
```

Comment:
@hermanschaaf I wonder if we need to return `net.IPNet` instead (receivers work on top of the pointer, but we might as well consider it)

Reply:
Yeah, I think because `Append(v net.IPNet)` on the builder takes a non-pointer, we can be consistent and also return the same here. The fields of `net.IPNet` are slices so there should be very little difference performance-wise.

Reply:
Decided to accept pointer in Append instead (as the receiver for IPNet is pointer).
Done in 63de964